tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -215,12 +286,26 @@ describe('Lobby - P0 Perspective', () => {
       cy.get('[data-cy=my-indicator]').contains(myUser.username);
       cy.get('[data-cy=opponent-indicator]').should('contain', opponentOne.username);
       cy.readyOpponent();
-      cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-ready-card"]').should('exist');
-
+      // openent's ready status should be true in the gameStore
+      cy.window()
+        .its('cuttle.gameStore')
+        .then((game) => {
+          expect(game.opponentIsReady).to.eq(true);
+        });
+      // opponent : ready class
+      cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-card-container"]').should('have.class', 'ready');
       cy.reload();
-      cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-ready-card"]').should('exist');
-      cy.get('[data-cy=my-indicator]').find('[data-cy="lobby-back-card"]').should('exist');
-
+      // openent's ready status should be true in the gameStore
+      cy.wait(1000); // fails without this wait
```

Comment:
Generally `cy.wait()` smells that something is wrong with the app or the test. One issue with `cy.wait()` is that it extends the test longer than we want in the case that things run quicker. It's generally preferable instead to use retryable assertions with timeouts so that cypress will keep trying until the assertion passes, within a specified time interval (default is 4 seconds).

That said, it's surprising that this specific `cy.wait()` would be necessary because the assertion before it,
```
cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-card-container"]').should('have.class', 'ready');
```

Should only be able to pass once the `opponentIsReady` value is set to true in the store because that's what the component is reading from. So I'd like to try removing this and confirming whether it's still necessary and if so, seeing what we can do to troubleshoot it
```suggestion
```